framework/Steps/IOStep.py

```diff
@@ -121,6 +121,11 @@ def _localInitializeStep(self,inDictionary):
       elif isinstance(inDictionary['Input'][i], (Models.ROM, Models.ExternalModel)):
         # ... file
         if isinstance(outputs[i],Files.File):
+          #XXX switching on the file extension is not very
+          # extendable.  Possibly should have a type or something
+          #Past condition: if 'py' == outputs[i].getExt().lower():
```

Comment:
This comment is now obsolete since the code has been fixed.

Reply:
Deleted comment.